akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowDelaySpec.scala

```diff
@@ -219,5 +219,17 @@ class FlowDelaySpec extends StreamSpec {
 
       probe.request(10).expectNextN(1 to 2).expectComplete()
     }
+
+    // repeater for #27095
+    "not throw NPE when using EmitEarly and buffer is full" taggedAs TimingTest in {
+      val result =
+        Source(1 to 9)
+          .delay(1.second, DelayOverflowStrategy.emitEarly)
```

Comment:
does it take 9 seconds to run this test? can we speed it up?

Reply:
No, since it uses `emitEarly` each new element causes the previous element to be emitted right away. Only the last element will get a 1s delay.